components/common/day_picker_navbar/index.tsx

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import { NavbarElementProps } from 'react-day-picker';
+
+interface CustomNavbarProps extends NavbarElementProps {
+    ifSearchDateSuggestion: boolean;
+}
+
+const Navbar: React.FC<Partial<CustomNavbarProps>> = (navbarProps: Partial<CustomNavbarProps>) => {
+    const {
+        onPreviousClick,
+        onNextClick,
+        className,
+        ifSearchDateSuggestion,
+    } = navbarProps;
+    const styleLeft: React.CSSProperties = {
+        float: 'left',
+        fontSize: 18,
+    };
+    const styleRight: React.CSSProperties = {
+        float: 'right',
+        fontSize: 18,
+    };
+
+    return (
+        <div className={className}>
+            <button
+                className='style--none'
+                style={styleLeft}
+                onClick={(e) => {
+                    e.preventDefault();
+                    if (onPreviousClick) {
+                        onPreviousClick();
+                    }
+                }}
+            >
+                <i
+                    className='fa fa-angle-left'
+                    aria-hidden='true'
+                    style={ifSearchDateSuggestion ? {marginLeft:48} : {}}
+                />
+            </button>
+            <button
+                className='style--none'
+                style={styleRight}
```

Comment:
same comment for the aria label. Also please check in the by running the app with voice over or other a11y tools if they are read out correctly.